README.md

```diff
@@ -130,6 +130,41 @@ sbot.metafeeds.findOrCreate((err, root) => {
 
 ## API
 
+### `sbot.metafeeds.v1.findOrCreate(details, cb)`
```

Comment:
I expect this to be one of the most-used functions, so it feels weird nesting it under v1 like this.
To what degree do we want the final API to lower level methods, to help guide people to use "sensible defaults" like this one. Is this a sensible default or just following the spec? (we still want advanced uses to be able to make arbitrary nesting I guess right?)

Reply:
keeping with the theme of high-level sensible default, i've dropped the `isFeed` function from this signature. hmmm

Reply:
I'm not sure about the removal of `isFeed`. I'm warming up to removing it, but part of me intuitively feels we might need it.

Reply:
I can't remember what we spoke when we were designing this in the NGI pointer grant.

But think of it from the perspective of a new developer who has little context for all these tools, and they see the field `feedpurpose`, I think it's reasonable that they may think "I can have two feeds with the same purpose".

Since in the metafeed tree there are going to be subfeeds generated by different app developers, it's reasonable to assume that these developers may be stepping on each other's toes, and that's why they might need more granularity, with `visit`